projects/cds/src/profiletag/services/profile-tag.injector.spec.ts

```diff
@@ -12,8 +12,8 @@ import {
   WindowRef,
 } from '@spartacus/core';
 import { BehaviorSubject } from 'rxjs';
-import { CdsConfig } from '../../config/cds-config';
-import { ProfileTagWindowObject } from '../model/profile-tag.model';
+import { CdsConfig } from '../../config/index';
```

Comment:
For some reason I couldn't reply in the thread: https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/5508#discussion_r350027319, so replying here:
True. But, when it comes to the Angular CLI production build, it produces either a warning or an error, depends on the case.

Feel free to resolve this conversation.